### DISCUSSION TITLE
Release Plan for v1.9.0

### DISCUSSION THREAD

**Discussion by kyleconroy** [sqlc-dev/sqlc#1066] [Issue Triage]
Release [v1.9.0](fhttps://github.com/kyleconroy/sqlc/milestone/10) will be published next Tuesday, July 6th. The headline features of this release are [view support](https://github.com/kyleconroy/sqlc/issues/193) and [pgx/pool](https://github.com/kyleconroy/sqlc/issues/472). As always, there are various bugs and features.

In preparation for the release, please test against main and report any issues. Thanks!

**Comment by fr3fou** [comment]
Was the release delayed?

**Reply by kyleconroy** [reply]
Yep. There are a few bugs on tip that I'm not comfortable releasing.

**Comment by kevinburke1** [comment]
This is the one issue we're seeing on tip - https://github.com/kyleconroy/sqlc/issues/1096

**Reply by fr3fou** [reply]
I really hope these issues also get fixed https://github.com/kyleconroy/sqlc/issues/1042 and https://github.com/kyleconroy/sqlc/pull/984

**Comment by kevinburke1** [comment]
Could you maybe tag the current commit as e.g. "1.9.0rc1" or "1.9.0beta1" or something?

**Reply by kyleconroy** [reply]
The release infrastructure isn't great at handling this. Can you use the `devel` Docker tag? 

If not, I can figure out how to cut a beta release without causing Homebrew to release a new version.

**Comment by kyleconroy** [comment]
I've updated the milestone. There are still quite a few blocking bugs, but I am going to release the current tip as `v1.9.0-beta1`.
